packages/hash/api/src/graphql/typeDefs/knowledge/page.typedef.ts

```diff
@@ -79,6 +79,10 @@ export const knowledgePageTypedef = gql`
     Get a page by its entity id.
     """
     knowledgePage(
+      """
+      The account ID that owns the Page.
+      """
+      ownedById: ID!
```

Comment:
Kind of an interesting point. Do we want to add these with the expectation that they will be needed? (which we've been doing in some other places if I recall) Or do we want to leave them out until we do need them.

Having them there does mean less changes later on, and forces us to think about some more things now, for example ensuring an Account ID is available on the codepaths of these functions (which can be important for making sure we have a `UserModel`, etc.)

Reply:
Although generally I don't like putting in code early on, because you just end up changing direction later on and forgetting you put these things in 🙃

Reply:
This is actually soemthing that is required by the following middleware: https://github.com/hashintel/hash/blob/dev/graph/packages/hash/api/src/graphql/resolvers/middlewares/canAccessAccount.ts#L24
Without this, you'd never have access to the page using that middleware